api/swagger.yaml

```diff
@@ -214,12 +214,14 @@ definitions:
           - `volume` a docker volume with the given `Name`.
           - `tmpfs` a `tmpfs`.
           - `npipe` a named pipe from the host into the container.
+          - `cluster` TODO add description
```

Comment:
Not sure exactly where the `Type` is used, and where `cluster` / `csi` can be used (do we check for incorrect types for these?)